packages/redux/src/checkout/actions/__tests__/fetchCharges.test.ts

```diff
@@ -15,6 +15,7 @@ describe('fetchCharges() action creator', () => {
   const checkoutMockStore = (state = {}) =>
```

Comment:
should we rename this file to `fetchCharge.test.ts`?

Reply:
done